sdk/search/azure-search-documents/README.md

```diff
@@ -21,7 +21,7 @@ The Azure Cognitive Search service is well suited for the following
 * In a search client application, implement query logic and user experiences
   similar to commercial web search engines.
 
-Use the Azure Search client library to:
+Use the Azure Cognitive Search client library to:
```

Comment:
Is this a confirmed change to add `Cognitive` in the Readme.
I remember @kaerm suggesting, Search isn't exactly a part of Cognitive umbrella..

Reply:
We had a readme meeting which aligned to use Cognitive Search everywhere.
https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/search/Azure.Search.Documents

Will bring this up to the search v-team meeting if there is document to refer.

Reply:
The title of the service is Cognitive Search so that's what should be used here, it's just not one of Cognitive Services